.env

```diff
@@ -1,5 +1,5 @@
 # General
-SATELLITE_NAME=UFO
```

Comment:
@chineduknight probably shouldn't change the default name of the satellite to yours, but doesn't affect much in the grand scheme of things :)